pkg/util/kubernetes/apiserver/metadata_controller_test.go

```diff
@@ -25,8 +25,6 @@ import (
 	gocache "github.com/patrickmn/go-cache"
 )
 
-func alwaysReady() bool { return true }
```

Comment:
Why are you removing this method ?

Reply:
To avoid having dependencies between two separate tests files